pyomo/contrib/incidence_analysis/tests/test_interface.py

```diff
@@ -1791,6 +1791,57 @@ def test_linear_only(self):
         self.assertIs(matching[m.eq2], m.x[2])
         self.assertIs(matching[m.eq3], m.x[3])
 
+    def test_add_edge(self):
```

Comment:
Can you add a test that implements the procedure we expect to follow for variable elimination? I.e.
```python
for adj_con in igraph.get_adjacent_to(var):
    for adj_var in igraph.get_adjacent_to(con):
        igraph.add_edge(adj_var, adj_con)
igraph.remove_nodes([var, con])
# Make sure total numbers of variables, constraints, and edges,
# as well as the adjacency pattern, are as we expect.
```